userguide/platform/userguide_deployment.adoc

```diff
@@ -357,8 +357,8 @@ To integrate JMX beans with Nagios, download the plugin from https://github.com/
 ./check_jmx_ng -v -U service:jmx:rmi:///jndi/rmi://127.0.0.1:8989/jmxrmi -O org.killbill.notificationq.api:name=NotificationQueueService -A NotificationProcessingSuspended -w false
 # Generic Kill Bill healthcheck, checks the overall state of the application (warns if unhealthy)
 ./check_jmx_ng -v -U service:jmx:rmi:///jndi/rmi://127.0.0.1:8989/jmxrmi -O org.killbill.billing.server.healthchecks:name=KillbillHealthcheck -A Healthy -w true
-# Monitors the size of the notification queue. Warning and Critical alerts often mean an overload of the system
-./check_jmx_ng -v -U service:jmx:rmi:///jndi/rmi://127.0.0.1:8989/jmxrmi -O metrics:name=org.killbill.notificationq.NotificationQueueDispatcher.pending-notifications -A Value -w 50 -c 100
+# Generates the overdue queue processing times like a histogram(max,min,average,mean,50th percentile,75th percentile,99th percentile etc.)
+./check_jmx_ng -v -U service:jmx:rmi:///jndi/rmi://127.0.0.1:8989/jmxrmi -O metrics:name=org.killbill.notificationq.NotificationQueueDispatcher.ove-overdue-check-queue-ProcessingTime -A Value -w 50 -c 100
```

Comment:
I'm not sure this is correct, also the [nagios-jmx-plugin ](https://github.com/killbill/nagios-jmx-plugin) plugin is really old and has not been updated recently, so I think the entire [nagios_integration](https://docs.killbill.io/latest/userguide_deployment#_nagios_integration section) can be removed. Thoughts @pierre ?

Reply:
I agree, we can probably remove the nagios section.